### PR TITLE
Yash_Assignment_22_45mins_Done

### PR DESCRIPTION
**Pull request by YashCJun20**
Please find the output

Program 1 :
image

Program2:
image

### CODE REVIEW

File: src/yashCJun20/Assignment22/VerifyPalindromeNumber.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+			else
+				System.out.println("Given number " + originalNumber+" is not Palindrome");
+	        }
+	public static void main (String[] args){
+		
+		VerifyPalindromeNumber verifyPalindromeNumber = new VerifyPalindromeNumber();
+		verifyPalindromeNumber.toVerifyPalindromeNumberOrNot(45654);
+		verifyPalindromeNumber.toVerifyPalindromeNumberOrNot(56789);
+		}
+	}
```
Comment (ApurvaJun20) [inline src/yashCJun20/Assignment22/VerifyPalindromeNumber.java:25]: Code you have written is good just check the extra spaces.

File: src/yashCJun20/Assignment22/VerifyPalindromeString.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+	         System.out.println("Given String "+name+" is not a Palindrome");
+	 
+	   }
+		public static void main(String [] args) {
+			VerifyPalindromeString verifyPalindromeString = new VerifyPalindromeString();
+			verifyPalindromeString.toVerifyPalindromeString("radar");
+			verifyPalindromeString.toVerifyPalindromeString("technocredits");
+		}
+	
+}
```
Comment (ApurvaJun20) [inline src/yashCJun20/Assignment22/VerifyPalindromeString.java:20]: Good work ..please add output also as an attachment from next assignment

Comment (YashCJun20) [reply src/yashCJun20/Assignment22/VerifyPalindromeString.java:20]: I have already attached output at the top

File: src/yashCJun20/Assignment22/VerifyPalindromeNumber.java (hunk 1 above)
Comment (YashCJun20) [reply src/yashCJun20/Assignment22/VerifyPalindromeNumber.java:25]: Sure

File: src/yashCJun20/Assignment22/VerifyPalindromeNumber.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
+package yashCJun20.Assignment22;
+class VerifyPalindromeNumber{
+	void toVerifyPalindromeNumberOrNot(int num){
+		int remender;
+		int reverseNumber=0;
+		int originalNumber;
```
Comment (KrishnaKTechnocredits) [inline src/yashCJun20/Assignment22/VerifyPalindromeNumber.java:6]: Directly initialize the value to the local variable. 
int originalNumber = num;

File: src/yashCJun20/Assignment22/VerifyPalindromeNumber.java [hunk 4]
```diff
@@ -0,0 +1,25 @@
+package yashCJun20.Assignment22;
+class VerifyPalindromeNumber{
+	void toVerifyPalindromeNumberOrNot(int num){
+		int remender;
```
Comment (KrishnaKTechnocredits) [inline src/yashCJun20/Assignment22/VerifyPalindromeNumber.java:4]: No need of this variable. Before use any local variable you should initialise it.

File: src/yashCJun20/Assignment22/VerifyPalindromeNumber.java [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+		int reverseNumber=0;
+		int originalNumber;
+		originalNumber=num;
+		while (num!=0)
+			{
+				remender=num%10;
+				reverseNumber=reverseNumber*10+remender;
+				num=num/10;
```
Comment (KrishnaKTechnocredits) [inline src/yashCJun20/Assignment22/VerifyPalindromeNumber.java:12]: reverseNumber=(reverseNumber*10) + (num%10);
num=num/10;

Use proper brackets when you are performing multiple operations.

File: src/yashCJun20/Assignment22/VerifyPalindromeNumber.java (hunk 1 above)
Comment (ApurvaJun20) [reply src/yashCJun20/Assignment22/VerifyPalindromeNumber.java:25]: sorry. I guess I missed the attachment.

**Pull request merged**